regression/tests/common/test_html_component.py

```diff
@@ -92,7 +95,7 @@ def test_word_cloud_advanced_component(self):
 
         # View Live
         self.unit_container_page.view_live_version()
-        self.lms_courseware.wait_for_page()
```

Comment:
I get the change above, but why did this need to be removed?

Reply:
I don't know, but I saw that the page was hanging on the wait, even though everything was loaded. I assume it's timing related?